sender.go

```diff
@@ -0,0 +1,37 @@
+package jshapi
+
+import (
+	"net/http"
+
+	"github.com/derekdowling/go-json-spec-handler"
+	"golang.org/x/net/context"
+)
+
+/*
+Sender is a function type definition that allows consumers to customize how they
+send and log API responses.
+*/
+type Sender func(context.Context, http.ResponseWriter, *http.Request, jsh.Sendable)
+
+/*
+SendHandler is what JSHAPI uses to send all API responses. This can be overidden to
+provide custom send or logging functionality.
+*/
+var SendHandler Sender = DefaultSender
+
+/*
+DefaultSender is the default sender that will log 5XX errors that it encounters
+in the process of sending a response.
+*/
+func DefaultSender(ctx context.Context, w http.ResponseWriter, r *http.Request, sendable jsh.Sendable) {
+
+	sendableError, isType := sendable.(jsh.ErrorType)
+	if isType && sendableError.StatusCode() >= 500 {
```

Comment:
Was able to remove a bunch of this logic by putting it here where it belongs in actuality: https://github.com/derekdowling/go-json-spec-handler/commit/a077109d98f6d6223555c633af4b04f9b7a3e3ec